### PR TITLE
Added info on virtualenv and setup.py in CONTRIBUTING.MD

### PR DESCRIPTION
**Pull request by lemmatum**
Added directions on how to setup virtualenv, install plasmapy dependencies, and setup plasmapy development files within virtualenv. Some of this is Linux specific.

This addresses issue #151

### CODE REVIEW

**Comment by lemmatum** [comment]
Feel free to include directions on setting up conda's version of a virtual environment under the "Virtual Environments" section, as i have no clue how to do that stuff!

**Comment by lemmatum** [comment]
May need to check for overlaps/conflicts with #160

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -80,6 +80,42 @@ git by going through these tutorials or a [Software
 Carpentry](https://software-carpentry.org/) workshop prior to making
 code contributions.
 
+### Virtual Environments
+
+Before you grab plasmapy from github, you are going to want to setup a sensible directory structure and a virtual environment. The virtual environment will allow you to import, run, and test your development version of plasmapy without contaminating or conflicting with other version of plasmapy or other packages that may be on your system.
```
Comment (namurphy) [inline CONTRIBUTING.md]: I'd suggest changing "plasmapy" to "PlasmaPy" and "github" to "GitHub" for consistency with the rest of the document.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -80,6 +80,42 @@ git by going through these tutorials or a [Software
...
+
+If you are running the Anaconda suite and want to use virtualenv to setup your virtual environment, you will have to let the system know where the Python compiler can be found. On linux this is done with (for example):
+
+```ShellSession
+export LD_LIBRARY_PATH="$HOME/anaconda3/lib/"
+```
+Exporting the library path to the dynamic linker will only last for the duration of the current shell session.
+
+Next you should create a sensible direction structure. Something like:
```
Comment (namurphy) [inline CONTRIBUTING.md]: direction --> directory

File: CONTRIBUTING.md (hunk 1 above)
Comment (lemmatum) [reply CONTRIBUTING.md]: done

File: CONTRIBUTING.md (hunk 2 above)
Comment (lemmatum) [reply CONTRIBUTING.md]: done

**Comment by lemmatum** [comment]
@StanczakDominik , cool, just be careful as plasmapy is lowercase is some places because that's the name of the directory I suggested to create for the virtualenv and it should be distinguished from PlasmaPy the package.

**Comment by lemmatum** [comment]
Looks good to me.
(and apparently review is only for reviewing all the changes and not specific commits 👅 , always learning!)

**Comment by StanczakDominik** [comment]
Pushed my changes, take a look @lemmatum :)

**Pull request merged**